Eplant/views/ChromosomeViewer/Viewer/GeneList.tsx

```diff
@@ -0,0 +1,140 @@
+// -------
+// IMPORTS
+// -------
+import React, { FC, useEffect, useState } from 'react'
+
+import List from '@mui/material/List'
+import ListItem from '@mui/material/ListItem'
+import ListItemButton from '@mui/material/ListItemButton'
+import ListItemIcon from '@mui/material/ListItemIcon'
+import ListItemText from '@mui/material/ListItemText'
+import useTheme from '@mui/material/styles/useTheme'
+
+import { GeneIcon } from '../icons'
+import { GeneItem } from '../types'
+
+import GeneInfoPopup from './GeneInfoPopup'
+
+// TYPES
+interface GeneListProps {
+  id: string
+  start: number
+  end: number
+  anchorOrigin: number[]
+}
+
+//----------
+// COMPONENT
+//----------
+const GeneList: FC<GeneListProps> = ({ id, start, end, anchorOrigin }) => {
+  // gene list
+  const [geneList, setGeneList] = useState<GeneItem[]>([
```

Comment:
I think you can just initialize this to an empty array, else it can be a bit confusing to debug in the future where this list contains some dummy data.